### ISSUE TITLE
A tutorial would be great, or is it just me needing more examples

### ISSUE THREAD

**Issue by byezy** [Algocircle/Cascadia.jl#27]
Following the brief readme, I get stuck, tried a few things, but I'm missing something

![image](https://user-images.githubusercontent.com/4742942/198857553-19a330e4-702f-4b79-b9eb-28f1f6c0db8a.png)

The element I'm trying to select by id is there:
![image](https://user-images.githubusercontent.com/4742942/198857594-93581e62-fd84-4d95-a11b-0e320e401c24.png)

I can't understand why the code can't find it...

Great package though, good work!

**Comment by aviks** [comment]
Yeah, I expect a selector like `#SensorNum4` to work. Can you possibly attach the html file that you are trying to parse, so that I can test this myself. 

Alternatively, try the [command line version of the go package](https://github.com/suntong/cascadia), and check if this julia package provides the same output as the go version. It is a bug if they do not produce the same result.

**Comment by byezy** [comment]
Hi thanks. Here is the html, less all the css + js guff
[Details.zip](https://github.com/Algocircle/Cascadia.jl/files/9916485/Details.zip)
 guff.

**Comment by aviks** [comment]
This does seem to work correctly for me

<img width="755" alt="image" src="https://user-images.githubusercontent.com/378918/199804244-5285f655-3a57-4640-98aa-df897e9f8c90.png">
